src/utils.ts

```diff
@@ -93,4 +94,18 @@ export function makeRandomHexString(length) {
 		result += chars[idx];
 	}
 	return result;
-}
\ No newline at end of file
+}
+
+export function deleteNoFailRecursive(path:any) {
+	if (fs.existsSync(path)) {
```

Comment:
what do you think to use a library for that such as https://www.npmjs.com/package/del ? I think it will make the code simpler to read

Reply:
Can definitely look at doing that - was just looking for something quick & dirty at first without adding more packages.